portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,61 +14,76 @@
 
 package com.google.sps.servlets;
 
-import java.util.List;
-import java.util.ArrayList;
-import com.google.gson.Gson;
-import com.google.sps.data.Comment;
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.Query.SortDirection;
-import com.google.appengine.api.datastore.FetchOptions;
-import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.Arrays;
+import com.google.sps.data.CommentPersistHelper;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static List<Comment> comments = new ArrayList<>();
+  /** Default number of comments to send */
+  private static final int DEFAULT_COMMENT_NUM = 25;
+  /** Query string which contains the number of comments to send */
+  private static final String NUM_COMMENTS_QUERY = "num-comments";
+  /** Header containing the total number of comments stored */
+  private static final String TOTAL_NUMBER_HEADER = "num-comments";
+  /** Query string that indicates whether to sort ascending or descending */
+  private static final String SORT_ASCENDING_QUERY = "sort-ascending";
+  /** Parameter which contains the ID's of the comments to delete */
+  private static final String DELETE_PARAMETER = "delete";
+
+  private static CommentPersistHelper comments;
```

Comment:
nit: the name for this is a bit confusing because comments is often used in this code to refer to a list of comments